fixtures/mock-oauth2-provider/package.json

```diff
@@ -47,5 +48,8 @@
   "devDependencies": {
     "@loopback/build": "^5.4.1",
     "@loopback/eslint-config": "^7.0.1"
+  },
+  "publishConfig": {
```

Comment:
Can you please move this toward the top of the file, for consistency with other package.json files?